common-arcana.lic

```diff
@@ -3,7 +3,7 @@
   Documentation: https://elanthipedia.play.net/Lich_script_development#common-arcana
 =end
 
-custom_require.call(%w[common events spellmonitor drinfomon])
+custom_require.call(%w[common common-items events spellmonitor drinfomon])
```

Comment:
This will break the scripts. I tried calling common-items inside here and another PR was attempted recently that broke everyone.

Reply:
That's why the references in common-items were removed (by moving the travel methods to common-travel). The break was caused by the circular dependency, which this fixes.